### PR TITLE
Balancing Borers

### PR DESCRIPTION
**Pull request by ghost**
Random loss of control based on brainloss removed. Borer stun reduced from 4 -> 3. Borer Reproduce Chems required doubled from 100 -> 200. Borer Event weight reduced from 20 -> 15. Borers no longer get forced to be unhidden when infesting someone as that made borers extremely obvious and ruined any chance at stealth borering.

This is due to concerns from some players and admins about borers stealing the show during cult rounds and generally getting out of control too fast due to needing only ~50 seconds to reproduce.

:cl:
del: Borers no longer randomly lose control based on host brain damage
tweak: Borer Dominate Victim stun time reduced from 4 -> 2.
tweak: Borers no longer force unhidden when infesting someone.
tweak: Borer event is rarer (weight 20->15).
tweak: Borer reproduction chemicals required increased from 100 to 200.
/:cl:

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
So... Borers can now only ever infest AFK/critical players?

**Comment by Durkel** [comment]
This doesn't really fix anything with them being either heal sluts making the crew near immortal, or just going around stabbing people like a asshole. If I have no reason to do bad guy stuff, shitting out a slug every two minutes to be nearly immortal isn't much of a trade off.

**Comment by uraniummeltdown** [comment]
I'll change the stun to 3 instead of 2, also borers aren't really a proper antag like changelings, just a small side-event that can appear. Also since borers don't get autorevealed when infesting, you could hide under a table and crawl into anyone who stands next to it.

**Comment by TehPear** [comment]
When I played borer I actually never used the stun to infest actual players. I just found someone standing idle not paying attention and infested them. Honk.

**Comment by jughu** [comment]
They are only immortal if you stop hitting SO KEEP HITTING THAT CRITICAL BODY

**Comment by AdamElTablawy** [comment]
@Durkel 

the entire point is that there ISN'T a downside to having them

They aren't antags, jesus christ

They're just useful little slug pals who's number one purpose is reproduction and that's it.

There's no reason NOT to have a borer if you aren't going to kill anyone, because it will heal you.

The main and only conflict that will come from borers is A. People roleplaying not wanting a slug in their head and chugging sugar, B. Borers forcefully infecting crew instead of being passive heal friends or C. Killing a borer host and being marked an enemy of a borer.

There's some confusion for some reason that borers are supposed to have innate downsides to having one or otherwise 'passive heal slut' or something.

No. There is no innate downside to having a borer-They reproduce and that's it. That's their purpose. They heal you, they reproduce, done.

The conflict with non-griefer borers comes from like I said, and most likely, killing a borer host and running into the borer not wanting you to do that, and forcing you not to.



tl;dr get rid of this stupid assumption that 'there's no reason not to have a borer' is a bad thing because it's not, that's the whole thing. There isn't a mechanical reason not to have a borer. The reason not to have a borer plays entirely on the fact that if a borer sees you killing a borer host it might try to stop you, which creates the conflict of you being a traitor with a borer and your target having a borer and the borer forcing you to disarm yourself so as not to kill the host, or forcing the captain to order the arrest and execution of the borer-killing HoS, etc.
